fs/filter/filter.go

```diff
@@ -191,7 +191,13 @@ func NewFilter(opt *Opt) (f *Filter, err error) {
 			return nil, err
 		}
 	}
+
+	inActive := f.InActive()
 	for _, rule := range f.Opt.FilesFrom {
+		if !inActive {
+			return nil, fmt.Errorf("The usage of --files-from overrides all other filters, it should be used alone")
+
```

Comment:
Extra new line

Reply:
Yes, I will remove it. No need for that.